products/kolibri.md

```diff
@@ -0,0 +1,168 @@
+releases to balance innovation with stability for enterprise deployments.
+
+### Support Types
+
+| Type | Duration | Description |
+|------|----------|-------------|
+| **STS** | 15 months | Short-Term Support releases receive active development with new features, improvements, and all bug fixes. Recommended for projects that want the latest capabilities. |
+| **LTS** | 36 months | Long-Term Support releases receive critical security fixes and important bug fixes only. Recommended for production environments requiring maximum stability. |
+
+### Version Status Legend
+
+| Symbol | Status | Description |
+|--------|--------|-------------|
+| 🟢 | **Active** | Current STS release with full feature development and support |
+| 🔵 | **LTS** | Long-term support with security and critical fixes only |
+| 🟡 | **Security** | Security fixes only, no new features or bug fixes |
+| 🔴 | **EOL** | End-of-life, no further updates |
+
+### Current Release Status
+
+| Version | Type | Status | Release | Period | End-of-Support |
+|---------|------|--------|---------|--------|----------------|
+| 5.x | STS | 🟢 Planned | Dec 2026 | 15m | Dec 2027 |
+| 4.x | LTS | 🟢 Active | Dec 2025 | 3y | Dec 2028 |
+| 3.x | STS | 🟡 Security | Dec 2024 | 15m | Dec 2025 |
+| 2.x | LTS | 🔵 LTS | Dec 2023 | 3y | Dec 2026 |
+| 1.x | LTS | 🔴 EOL | Dec 2021 | 3y | Dec 2024 |
+| 0.x | Initial | 🔴 EOL | Jul 2020 | - | Dec 2021 |
+
+## Release Cycle
+
+KoliBri regularly publishes minor and patch releases with new features, bug fixes, and security
+updates. Major releases occur approximately once a year and may contain breaking changes.
+
+- **Version 5.x** 🟢 (STS, planned): Next short-term release scheduled for December 2026.
+- **Version 4.x** 🟢 (LTS): Active development with new features, enhanced theming system,
+  improved performance, and modernized component architecture. Supported until December 2028.
+- **Version 3.x** 🟡 (STS): Security fixes only until December 2025.
+- **Version 2.x** 🔵 (LTS): Long-term support with security and critical bug fixes until December 2026.
+- **Version 1.x** 🔴 (EOL): End-of-life since December 2024.
+- **Version 0.x** 🔴 (EOL): Initial release, end-of-life since December 2021.
+
+## Framework Support
+
+KoliBri provides adapters for various frameworks:
+
+| Framework | Supported Versions |
+|-----------|-------------------|
+| Angular   | 16, 17, 18, 19, 20, 21 |
+| React     | 18, 19 |
+| Vue       | 3.x |
+| Preact    | 10.x |
+| Solid     | 1.x |
+| Svelte    | 4.x, 5.x |
+
+## Accessibility
+
+All KoliBri components are certified according to the following standards:
+
+- **WCAG 2.1** Level AA (Web Content Accessibility Guidelines)
+- **BITV 2.0** (German Accessible Information Technology Regulation)
+- **EN 301 549** (European Standard for Digital Accessibility)
+
+Components are regularly validated using axe-core and manual screen reader testing.
+
```

Comment:
This is information better documented at Kolibri docs. In case something important happens wrt Accessibility (say WCAG not being tested in a future release) -  we can add a note for that. 

```suggestion
```